data/human/frontier jobs.txt

```diff
@@ -237,16 +237,16 @@ mission "Instruct Novice Pilot"
 
 
 mission "Small Pirate gambling"
-	description "Survive the pirate attack!"
+	invisible
 	minor
-	repeat
 	to offer
 		random < 3
```

Comment:
```suggestion
		random < (3 - "Small Pirate gambling: failed" - "Small Pirate gambling: declined")
```
How about this? Then "repeat" can be added, but the mission would only repeat at most three times.

Reply:
Don't forget that you can do something like `repeat 3` to have it repeat a finite number of times

Reply:
Yes, but the original suggestion in 3125 was to have the mission become less likely upon each repeat. By just doing "repeat 3" the chance of the mission offering again stays the same until it stops offering.

Reply:
There isn't any reason for us to allow this mission to repeat: the only use case is to let the player replay the mission in the same save file, which is something we don't want anyway.

Reply:
From what I read, someone said making it less likely on each offer was sufficient, and there were two reasons for not making it not repeatable at all, one of which was was that nobody implemented the functionality of making it less likely on each offer, so I implemented it. Up until 45 minutes ago, the only other reason for making it not repeatable at all was that this PR reinforced it (circular reasoning). Now that you have edited your description, it looks like the issue you have is that the PR has "distinctive text" and you got a different PR merged that made it so that a mission with distinctive text couldn't repeat. This sentiment stems from a comment by MZ stating that it is annoying to see the same text over and over again, and tehhowch explained that this issue was resolved by making it less likely, going back to where we are.

The main reason for allowing this mission to repeat is, as Amazinite said, "the random nature of this mission." However, Amazinite then said that having to make a new pilot wasn't necesarily a bad thing. So, I think that with the new information I have received, I am approving this change as it doesn't matter either way. The phrase "this should be sufficient" to me implied that we should do that rather than something else.